### DISCUSSION TITLE
External Authentication  How do I get started?  ！！！！

### DISCUSSION THREAD

**Discussion by chinasqzl** [OrchardCMS/OrchardCore#17188] [Q&A]
![image](https://github.com/user-attachments/assets/c0daecc8-fda9-4481-b52c-43fb63fc1af4)

**Comment by Piedone** [comment]
See e.g. https://docs.orchardcore.net/en/latest/reference/modules/Microsoft.Authentication/. Do you have any specific question?

**Comment by chinasqzl** [comment]
Thank you
